system/HTTP/RedirectResponse.php

```diff
@@ -63,13 +63,14 @@ public function to(string $uri, int $code = null, string $method = 'auto')
 		// for better security.
 		if (strpos($uri, 'http') !== 0)
 		{
-			$url = current_url(true)->resolveRelativeURI($uri);
-			$uri = (string)$url;
+			$uri = (string) current_url(true)->resolveRelativeURI($uri);
 		}
 
 		return $this->redirect($uri, $method, $code);
 	}
 
+	//--------------------------------------------------------------------
```

Comment:
I'm not a fan of the separators between all functions. I think they serve some visual clarity as logical dividers between types of methods in large classes. We probably need to decide how we want to handle them framework-wide for consistency, but for now I think not adding them is better IMO.

Reply:
This is not my addition. But I'll remove them. Personally not a fan either.